### PR TITLE
MGMT-8890: Add 4.11 single-arch release image to defaults

### PR DESCRIPTION
**Pull request by mkowalski**
Contributes-to: [MGMT-8890](https://issues.redhat.com//browse/MGMT-8890)

### CODE REVIEW

File: data/default_release_images.json [hunk 1]
```diff
@@ -35,6 +35,25 @@
...
+      "version": "4.11.0",
+      "default": true
+    },
+    {
+      "openshift_version": "4.11",
+      "cpu_architecture": "arm64",
+      "cpu_architectures": [
+          "arm64"
+      ],
+      "url": "quay.io/openshift-release-dev/ocp-release:4.11.0-arm64",
```
Comment (osherdp) [inline data/default_release_images.json:54]:
```suggestion
      "url": "quay.io/openshift-release-dev/ocp-release:4.11.0-aarch64",
```

**Review by osherdp — CHANGES_REQUESTED** [review]

File: data/default_release_images.json (hunk 1 above)
Comment (mkowalski) [reply data/default_release_images.json]: Fixed, thanks!

**Review by osherdp — APPROVED** [review]

**Comment by osherdp** [comment]
/override ci/prow/edge-e2e-metal-assisted-day2-single-node
ignore, @eliorerz is fixing it

**Comment by osherdp** [comment]
/hold
Failure seems somewhat related to the changes of this PR

**Comment by mkowalski** [comment]
/unhold

Indeed that was the case; we didn't foresee shipping single-arch next to multi-arch so the tests were opinionated towards some logic implemented for "only multi-arch" scenario. As the files used by subsystem are directly coming from this repo and not from the separate test files, the test need to be adjusted. Didn't find an easy way to provide a separate configuration to the test so as long as our tests and prod consume the same file, one has to be opinionated ¯\_(ツ)_/¯

**Review by osherdp — APPROVED** [review]

**Comment by mkowalski** [comment]
/test edge-e2e-metal-assisted-day2-single-node

Seems to almost always fail, let's just retry

**Pull request merged**